build.psm1

```diff
@@ -468,6 +468,16 @@ cmd.exe /C cd /d "$location" "&" "$($vcVarsPath)\vcvarsall.bat" "$NativeHostArch
         }
         New-Item -Path $refDestFolder -ItemType Directory -Force -ErrorAction Stop > $null
         Copy-Item -Path $refAssemblies -Destination $refDestFolder -Force -ErrorAction Stop
+
+        # create the symlinks needed on Linux
```

Comment:
Could you please use the same comment from `New-UnixPackage` so that it's more clear that this is required by Linux remoting over OMI:
```
        # add two symbolic links to system shared libraries that libmi.so is dependent on to handle
        # platform specific changes. This is the only set of platforms needed for this currently
        # as Ubuntu has these specific library files in the platform and OSX builds for itself
        # against the correct versions.
```